src/app/admin/sidemenu/sidemenu.component.ts

```diff
@@ -112,6 +115,12 @@ activeMenu: string = '';
     }   
   }
 
+  show(menu){
+    menu == 'dashboard' ? this.showDashboard = true : this.showDashboard = false;
+    menu == 'roles' ? this.showRoles = true : this.showRoles = false;
```

Comment:
this.showDashboard  = (menu == 'dashboard' ) ?  true :   false;  use this ternary condition for below conditions too 
@suvarnakale

Reply:
Okay